src/views/GovernanceTaskList/index.tsx

```diff
@@ -76,7 +81,9 @@ function GovernanceTaskList() {
       showMessageOnNextPage(message, { type: 'success' });
       history.push('/');
     };
-    dispatch(archiveSystemIntake({ intakeId: systemId, redirect }));
+
+    await archive();
+    redirect();
```

Comment:
Nice, the only thing I could see here is maybe we could do some error handling.

Reply:
oh - doh! you are right - checking into this

Reply:
what could we do here on an error? @aterstriep

Reply:
or @adamodd

Reply:
Up at the top of page there are some Alert conditions. I'm assuming this error could follow that pattern. I'll look around the figmas for more some kind of slim alert error visual + text. We usually just say some error occurred and please try again.